github-actions/scan/src/post-scan.ts

```diff
@@ -20,8 +20,8 @@ export function downloadReports(formats: string[]): void {
         return;
     }
 
-    const jobUUID = getJobUUID();
-    core.debug(jobUUID);
+    const jobUUID = getFieldFromJsonReport('jobUUID');
+    core.debug('JobUUID: ' + jobUUID);
```

Comment:
We could load here the JSON report and hold the JSON object as a local variable.
Then we could use `getFieldFromJsonReport('jobUUID', json);` to fetch the job uuid.

At the end, we could return (instead of void) the json object - so it can be reused to call `reportOutputs(json)` later. And `reportOutput` method can use the parameter for `analyzeFindings(json)` method call + field fetching.

(Please read my next comment for `getFieldFromJsonReport` it is related) 
_Topic: reduce JSON reads/objects_